### DISCUSSION TITLE
AuditV2测试是在哪个文件？

### DISCUSSION THREAD

**Discussion by peixubin** [hhyo/Archery#2719] [Q&A]
文件sql/utils/test_workflow_audit.py 里面的测试方法好像在跑测试的时候并不运行

**Comment by LeoQuote** [comment]
运行的，是用 pytest 运行测试，详见 https://github.com/hhyo/Archery/wiki/development#%E5%8D%95%E5%85%83%E6%B5%8B%E8%AF%95

**Comment by peixubin** [comment]
那些fixture是在哪里定义的？ 比如sql_workflow。

**Reply by LeoQuote** [reply]
全部在这里，欢迎更新wiki https://github.com/hhyo/Archery/blob/master/conftest.py
